scripts/package-build

```diff
@@ -148,14 +148,16 @@ for pkgname in "${pkgnames[@]}"; do
             # Run packaging script
             package
 
-            section "Stripping $pkgname"
-            docker container run --interactive --rm \
-                --mount type=bind,src="$(realpath "$pkgdir")",dst=/pkg \
-                "$(image-name "$image")" /bin/bash \
-                << "SCRIPT"
+            if ! printf '%s\n' "${flags[@]}" | grep -qP '^nostrip$'; then
```

Comment:
I suggest using [the `has-element` helper](https://github.com/toltec-dev/toltec/blob/5a206fef9b369597eb348ed8dcd3d3ea6f9466f4/scripts/package-lib#L182) for this test.